.github/workflows/git-artifacts.yml

```diff
@@ -289,12 +373,27 @@ jobs:
           *" ${{matrix.artifact.name}} "*|*" ${{matrix.artifact.name}}-${{matrix.arch.name}} "*) ;; # build this artifact
           *) echo "SKIP=true" >>$GITHUB_ENV;;
           esac
+      - name: Configure user
+        shell: bash
+        run:
+          USER_NAME="${{github.actor}}" &&
+          USER_EMAIL="${{github.actor}}@users.noreply.github.com" &&
+          mkdir -p "$HOME" &&
+          git config --global user.name "$USER_NAME" &&
+          git config --global user.email "$USER_EMAIL" &&
+          echo "PACKAGER=$USER_NAME <$USER_EMAIL>" >>$GITHUB_ENV
       - name: Download pkg-${{matrix.arch.name}}
-        if: env.SKIP != 'true'
+        if: env.SKIP != 'true' && matrix.arch.name != 'arm64'
         uses: actions/download-artifact@v1
         with:
           name: pkg-${{matrix.arch.name}}
           path: pkg-${{matrix.arch.name}}
+      - name: Download pkg-i686
```

Comment:
Hrm. Maybe we should keep `arch.name == i686` for ARM64, for now, and only add a separate "coordinate" to indicate that we want to add the ARM64 artifacts? Something like `matrix.arm64 = true`, for simplicity?